iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/AbstractTreeDataImporter.java

```diff
@@ -54,6 +54,8 @@ public abstract class AbstractTreeDataImporter implements Importer {
 
     public static final String MISSING_PERMANENT_LIMIT_PERCENTAGE = "iidm.import.xml.missing-permanent-limit-percentage";
 
+    public static final String MINIMAL_VALIDATION_LEVEL = "iidm.import.xml.minimal-validation-level";
```

Comment:
import -> serde

Reply:
I don't know if it is a good idea to remove the "import" part of the parameter. We can have parameters dealing with the same concept for import and export and we may want them to have different values (for instance, we have both `iidm.export.xml.extensions` and `iidm.import.xml.extensions` parameters).

We can maybe use something like "serde.import", or just "import"?

Reply:
@flo-dup proposes `iidm.import.minimal-validation-level`

Reply:
The parameter was changed accordingly.